src/tests/test_python_module.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+
+def test_init(selenium_module_standalone):
```

Comment:
It would be preferable to use a module-scoped fixture for these so we don't reload it three times. These tests don't need to be isolated from each other. (The `test_init` in `test_python.py` really shouldn't use `selenium_standalone` either...)

Reply:
It turns out `test_init` uses `selenium_standalone` because the module scoped `selenium` fixture [clean logs](https://github.com/pyodide/pyodide/blob/main/conftest.py#L647) which can cause the test to fail -